plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/recipes/UpgradeJenkinsTestHarnessVersion.java

```diff
@@ -0,0 +1,84 @@
+package io.jenkins.tools.pluginmodernizer.core.recipes;
+
+import io.jenkins.tools.pluginmodernizer.core.model.JDK;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.maven.ChangePropertyValue;
+import org.openrewrite.maven.MavenIsoVisitor;
+import org.openrewrite.maven.RemoveRedundantDependencyVersions;
+import org.openrewrite.xml.tree.Xml;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Upgrade Jenkins Test Harness version to ensure compatibility with jenkins.version.
+ */
+public class UpgradeJenkinsTestHarnessVersion extends Recipe {
+    /**
+     * Logger
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(UpgradeJenkinsTestHarnessVersion.class);
+
+    /**
+     * The jenkins version.
+     */
+    @Option(displayName = "Version", description = "Jenkins version.", example = "2.440.3")
+    String jenkinsVersion;
+
+    /**
+     * Constructor.
+     * @param jenkinsVersion The Jenkins version.
+     */
+    public UpgradeJenkinsTestHarnessVersion(String jenkinsVersion) {
+        this.jenkinsVersion = jenkinsVersion;
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Upgrade Jenkins Test Harness version";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Upgrade Jenkins Test Harness version to ensure compatibility with jenkins.version.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return new MavenIsoVisitor<>() {
+
+            @Override
+            public Xml.Document visitDocument(Xml.Document document, ExecutionContext ctx) {
+
+                String latestCompatibleVersion = JDK.getLatestTestHarnessVersion(jenkinsVersion);
+
+                if (latestCompatibleVersion == null) {
+                    LOG.info(
+                            "No compatible Jenkins Test Harness version found for Jenkins version {},  Defaulting to RELEASE'.",
+                            jenkinsVersion);
+                    latestCompatibleVersion = "RELEASE";
```

Comment:
Shouldn't it be 'latest.release' 

​​https://docs.openrewrite.org/reference/dependency-version-selectors#supported-dependency-version-selectors